packs/nuxt.js

```diff
@@ -0,0 +1,23 @@
+const icon = `data:image/svg+xml,<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 124 124"><path fill-rule="evenodd" clip-rule="evenodd" d="M55.75 27.155c-3.222-5.54-11.278-5.54-14.5 0L6.134 87.535C2.912 93.075 6.94 100 13.384 100h27.413c-2.753-2.407-3.773-6.57-1.69-10.142L65.704 44.27 55.75 27.155Z" fill="#80EEC0"/><path d="M78 40.4c2.667-4.533 9.333-4.533 12 0L119.06 89.8c2.667 4.533-.666 10.199-5.999 10.199H54.938c-5.333 0-8.666-5.666-6-10.199L78 40.4Z" fill="#00DC82"/></svg>`;
```

Comment:
```suggestion
const icon = `data:image/svg+xml,<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 124 124"><path fill-rule="evenodd" clip-rule="evenodd" d="M55.75 27.155c-3.222-5.54-11.278-5.54-14.5 0L6.134 87.535C2.912 93.075 6.94 100 13.384 100h27.413c-2.753-2.407-3.773-6.57-1.69-10.142L65.704 44.27 55.75 27.155Z" fill="%2380EEC0%22/%3E%3Cpath d=%22M78 40.4c2.667-4.533 9.333-4.533 12 0L119.06 89.8c2.667 4.533-.666 10.199-5.999 10.199H54.938c-5.333 0-8.666-5.666-6-10.199L78 40.4Z%22 fill=%22%2300DC82%22/%3E%3C/svg%3E`;
```

I dropped unnecessary space encoding and encoded the hashes so that copy/pasting this value in the address bar worked.